CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased 2.x]
 ### Added
-- [Extensions] Moving Extensions APIs to protobuf serialization. ([#6960](https://github.com/opensearch-project/OpenSearch/pull/6960))
```

Comment:
The [changelog](https://github.com/opensearch-project/OpenSearch/pull/7302/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR91) added in the PR raised against `main` is different here. We can add a new entry here for #7302

Reply:
I realized its not worth having multiple lines in change log, since its not really adding value for readers. 
I'll just link it to the meta issue. I'll fix the changelog in `main` in my next PR.

Reply:
Or you can add the PRs related to protobuf to the same changelog. Something like:
```
Moving Extensions APIs to support cross versions via protobuf. (#PR1, #PR2)